.github/workflows/breaking-change-to-main.yml

```
@@ -0,0 +1,22 @@
+name: Check for breaking changes targeting main
+on:
+  pull_request:
+    branches:
+      - "**"
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+  merge_group:
+    types: [checks_requested]
+
+jobs:
+  breaking-change-to-main:
+    name: Check for breaking changes targeting main
+    uses: CQCL/hugrverse-actions/.github/workflows/breaking-change-to-main.yml@luca/breakingprtmp
+    secrets:
+      GITHUB_PAT: ${{ secrets.HUGRBOT_PAT }}
+    with:
+      release_branch: release-rs-v0.16.0
```

I added the variable :)
```suggestion
      release_branch: ${{ vars.BREAKING_RELEASE_BRANCH }}
```